PR for SRVKE-1742: Document "Event Transformation" section for OpenShift Serverless Eventing (#98334)
openshift/openshift-docs: modules/serverless-event-transformation-examples.adoc

// Module included in the following assemblies:
//
// serverless/eventing/serverless-event-transformation.adoc

:_mod-docs-content-type: REFERENCE
[id="serverless-event-transformation-examples_{context}"]
= Event transformation examples

> The following examples shows how you can use JSONata in `EventTransform` resources to reshape and enrich events for common business scenarios.

Reviewer: ```suggestion
You can use these examples to apply JSONata in `EventTransform` resources to reshape and enrich events for common business scenarios.
```
